public/docs/_examples/intro/dart/lib/logger_service.dart

```diff
@@ -0,0 +1,20 @@
+library developer_guide_intro.logger_service;
+
+import 'dart:html';
+
+import 'package:angular2/angular2.dart';
+
+@Injectable()
+class Logger {
+  void log(dynamic msg) {
+    window.console.log(msg);
```

Comment:
It looks like you could just use print() instead. Also, we don't usually use dynamic in signatures. Object would do.

```
void log(Object msg) {
  print(msg);
}